src/routes/Plugin/Common/Rule.js

```diff
@@ -38,6 +38,10 @@ class AddModal extends Component {
     });
 
     this.state.ruleConditions = ruleConditions;
+
+    this.initDic("Operator");
+    this.initDic("MatchMode");
+    this.initDic("paramType");
```

Comment:
Can we keep consistent with other type and use `ParamType` instead of `paramType`. Use a `prefix` is more better. WDYT? @fengzhenbing

Reply:
Since there is no uniform case in the dictionary table now, I will update the shenyu‘s schema data. After that,I will use lowercase consistently next step.